lang/Polski.json

```diff
@@ -138,9 +138,14 @@
                 "autostart": "Autostart: ",
                 "tray": "Autostart z zasobnika: ",
                 "tray_notifications": "Powiadomienia z zasobnika: ",
-                "priority_only": "Tylko priorytetowe: ",
+                "priority_mode": "Tryb priorytetu: ",
                 "proxy": "Proxy (wymaga restartu):"
             },
+            "priority_modes": {
+                "priority_only": "Tylko lista priorytetowa",
+                "ending_soonest": "Najwcześniejszy koniec",
+                "low_availability": "Najpierw niska dostępność"
+            },
```

Comment:
```suggestion
            "priority_modes": {
                "priority_only": "Tylko lista priorytetów",
                "ending_soonest": "Kończące się najwcześniej",
                "low_availability": "Najniższa dostępność"
            },
```

Reply:
Yes, it's okay. It looks more natural. I forgot to change **Najpierw niska dostępność** to **Niska dostępność**. **Najniższa dostępność** is also okay.